### ISSUE TITLE
tests fail on 1.0 due to NamedTuples change

### ISSUE THREAD

**Issue by Sacha0** [mauro3/Parameters.jl#76]
```
julia> versioninfo()
usJulia Version 1.0.0
Commit 5d4eaca0c9* (2018-08-08 20:58 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin17.7.0)
  CPU: Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)

julia> using Parameters

(v1.0) pkg> test Parameters
   Testing Parameters
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package NamedTuples [73a701b4]:
 NamedTuples [73a701b4] log:
 ├─possible versions are: [0.0.1-0.0.3, 0.1.0, 1.0.0, 2.0.0, 2.1.0, 3.0.0-3.0.3, 4.0.0-4.0.2] or uninstalled
 ├─restricted to versions 0.0.0-* by an explicit requirement, leaving only versions [0.0.1-0.0.3, 0.1.0, 1.0.0, 2.0.0, 2.1.0, 3.0.0-3.0.3, 4.0.0-4.0.2]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```
I conjecture that the combination of [this unconditional REQUIRE line](https://github.com/mauro3/Parameters.jl/blob/49ad1e1c295aa6b65681fa6abdddf9657553250e/test/REQUIRE#L2) and the [recent change to NamedTuples.jl that prevents installation on 1.0](https://github.com/JuliaLang/METADATA.jl/pull/17104) cause this issue. (cc @simonbyrne) Best!

**Comment by mauro3** [comment]
Thanks Sacha! I'll just drop Julia 0.6 (the last commit on master).  I will tag once that is through.

**Comment by mauro3** [comment]
x-ref: https://github.com/JuliaLang/METADATA.jl/pull/17660

**Comment by mauro3** [comment]
Fixed on latest tag v0.10.0

**Comment by Sacha0** [comment]
Thanks again Mauro! :)
